src/org/ods/orchestration/Stage.groovy

```diff
@@ -71,6 +71,7 @@ class Stage {
                 def unstashStartTime = System.currentTimeMillis();
                 script.unstash("wholeWorkspace")
                 script.echo "Unstashing workspace took ${System.currentTimeMillis() - unstashStartTime}ms"
+                // should use GitCredentialStore but I am not sure how to test mros.
```

Comment:
Would be nice to do this as well. I can help with how to test this, at least how to do the manual test. Every run of the orchestration pipeline will call this as it's needed to push a commit to the remote.